catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java

```diff
@@ -203,6 +223,22 @@ private String parseSrc(String localSource) {
     return src;
   }
 
+  private void parseSrcs(String localSource) {
```

Comment:
✏️ Since `parseSrc` and `parseSrcs` both have side effects on the fields, it would be clearer to move this logic into their setters.